portal-api/src/scheduler/jobs/fetch-workflow-type-b.js

```diff
@@ -1,154 +0,0 @@
-const moment = require('moment');
```

Comment:
RE my last comment sorry i've just seen this - can we update the jira ticket and link to this PR?

Reply:
Also we could now remove some workflow things in `deal-status.controller.js`

Reply:
we can indeed! I need to find out exactly where that is 😄